sample_application/app.py

```diff
@@ -63,6 +63,10 @@ def create_app(blueprint_only=False):
         return blueprint
     app.register_blueprint(blueprint)
 
+    handler = setup_logging_handler(level=app.config['SAMPLE_APPLICATION_LOG_LEVEL'])
+    app.logger.addHandler(handler)
+    app.logger.setLevel(handler.level)
```

Comment:
Oddly, it would not work without line 68 for the blueprints, but works fine on my own app. I could not figure out what was overriding the log level set by line 66.